project/src/components/common/map.tsx

```diff
@@ -40,17 +31,17 @@ const currentCustomIcon = new Icon({
 
 type MapProps = {
   mapType: MapType;
-  city?: City;
+  cityLocation: City;
   points: Offer[];
   selectedPoint: Offer | undefined;
 };
 
-function Map({ mapType, city = MOCK_CITY, points, selectedPoint }: MapProps): JSX.Element {
+function Map({ mapType, cityLocation, points, selectedPoint }: MapProps): JSX.Element {
   const mapRef = useRef(null);
-  const map = useMap(mapRef, city);
+  const [ map, offersLayerGroup ] = useMap(mapRef, cityLocation);
```

Comment:
а зачем тут пробелы? 
```suggestion
  const [map, offersLayerGroup] = useMap(mapRef, cityLocation);
```